frontend/src/libs/components/form/form.tsx

```diff
@@ -18,6 +21,37 @@ type Properties<T extends FieldValues> = {
   onSubmit: (payload: T) => void;
 };
 
+const renderField = <T extends FieldValues = FieldValues>(
```

Comment:
Maybe put it in a separate hook to not enlarge the component? Many cases have not been processed yet, but there is already a lot of code.

Reply:
let's do it separately, not in this ticket @h0wter

Reply:
It's a good idea, but not for now, thanks.